spec/src/main/asciidoc/SOAPAttachments.adoc

```diff
@@ -128,8 +125,8 @@ image:soapatt-8.png[image]
 
 Abstract
 
-The SOAP with Attachments API for Java™ (SAAJ
-1.4) enables developers to produce and consume messages conforming to
+The Jakarta SOAP with Attachments API for Java (SAAJ) 
```

Comment:
This should be "The SOAP with Attachments API for Jakarta (SAAJ)"

Reply:
No, official name of this specification is `Jakarta SOAP with Attachments specification` and scope statement has been decided to be: `Jakarta SOAP with Attachments defines an API enabling developers to produce and consume messages conforming to the SOAP 1.1, SOAP 1.2, and SOAP Attachments Feature.` (see https://github.com/eclipse-ee4j/jax-ws-api/issues/57), so it should be changed to sth like that

Reply:
You are right, sorry. I didn't read all the paragraph